coap-gateway/service/signIn_test.go

```diff
@@ -101,8 +101,12 @@ func TestSignInDeviceSubscriptionHandler(t *testing.T) {
 	require.True(t, cancelCtx.Err() == context.Canceled)
 
 	co1 := testCoapDial(t, testCfg.GW_HOST, "")
-	_, code := runSignIn(t, CertIdentity, signUpResp, co1)
-	require.Equal(t, coapCodes.Unauthorized, code)
+	resp, err := doSignIn(t, CertIdentity, signUpResp, co1)
```

Comment:
Previously `runSignIn` called `co.ReleaseMessage(resp)`, this is now skipped by calling `doSignIn` directly. Is that OK?

Reply:
Yes, it's ok. the message just will be not stored in the cache for reuse.